inspirehep/records/api/literature.py

```diff
@@ -117,3 +123,8 @@ def add_files(self, documents=None, figures=None):
                 files.append(metadata)
         super().update(builder.record)
         return files
+
+    def _dump_for_es(self):
+        serialized_data = super()._dump_for_es()
+        serialized_data["_ui_display"] = json.dumps(self.get_ui_data())
```

Comment:
why isn't this part of the upstream class, if you're not redefining `get_ui_data` here? all formats will eventually have a `_ui_display` field.